Too many processes are started on systems with large number of threads

**Describe the bug**
Fresh install on debian bookworm.
Allocated 6GB ram to the VM (documentation says at least 4).
Quickly the system runs out of memory due to hundreds of python processes.

**Screenshots**
![image](https://github.com/minvws/nl-kat-coordination/assets/85709915/3a014123-8976-4706-ac0b-c6ce5fb8da91)

![image](https://github.com/minvws/nl-kat-coordination/assets/85709915/a0f55bd5-2adc-4265-86f6-98ef0cb0cc18)


**OpenKAT version**
v1.10.0

```
2023-08-05T22:05:49.918520+02:00 sec-scan kat-mula[618]: [2023-08-05 22:05:49 +0200] [618] [INFO] [h11_impl] [uvicorn.access] ::1:41556 - "GET /queues HTTP/1.1" 200
2023-08-05T22:05:49.923108+02:00 sec-scan kat-mula[618]: [2023-08-05 22:05:49 +0200] [618] [INFO] [h11_impl] [uvicorn.access] ::1:41556 - "POST /queues/boefje-fvh/pop HTTP/1.1" 200
2023-08-05T22:05:50.004112+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection] Connection closed while authenticating indicating a probable authentication error
2023-08-05T22:05:50.004368+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection] Connection closed while authenticating indicating a probable authentication error
2023-08-05T22:05:50.059740+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection_workflow] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2023-08-05T22:05:50.059879+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection_workflow] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2023-08-05T22:05:50.059951+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection_workflow] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 1 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - None.
2023-08-05T22:05:50.060002+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection_workflow] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 1 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - None
2023-08-05T22:05:50.060049+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection_workflow] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 1 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - None.
2023-08-05T22:05:50.060121+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [ERROR] [connection_workflow] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 1 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - None
2023-08-05T22:05:50.060231+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [blocking_connection] Connection workflow failed: AMQPConnectionWorkflowFailed: 1 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - None
2023-08-05T22:05:50.060289+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [ERROR] [blocking_connection] Connection workflow failed: AMQPConnectionWorkflowFailed: 1 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - None
2023-08-05T22:05:50.060713+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [blocking_connection] Error in _create_connection().
2023-08-05T22:05:50.060809+02:00 sec-scan kat-octopoes[731]: Traceback (most recent call last):
2023-08-05T22:05:50.060881+02:00 sec-scan kat-octopoes[731]:   File "/opt/venvs/kat-octopoes/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
2023-08-05T22:05:50.060950+02:00 sec-scan kat-octopoes[731]:     raise self._reap_last_connection_workflow_error(error)
2023-08-05T22:05:50.061011+02:00 sec-scan kat-octopoes[731]: pika.exceptions.ProbableAuthenticationError: ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2023-08-05T22:05:50.061094+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [ERROR] [blocking_connection] Error in _create_connection().#012Traceback (most recent call last):#012  File "/opt/venvs/kat-octopoes/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection#012    raise self._reap_last_connection_workflow_error(error)#012pika.exceptions.ProbableAuthenticationError: ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2023-08-05T22:05:50.062115+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [CRITICAL] [api] ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2023-08-05T22:05:50.062200+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [CRITICAL] [api] ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2023-08-05T22:05:50.062465+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [INFO] [h11_impl] 127.0.0.1:41238 - "GET /fvh/objects/random?amount=50&scan_level=1&scan_level=2&scan_level=3&scan_level=4 HTTP/1.1" 500
2023-08-05T22:05:50.062594+02:00 sec-scan [2023-08-05 22:05:50 +0200] [731] [INFO] [h11_impl] 127.0.0.1:41238 - "GET /fvh/objects/random?amount=50&scan_level=1&scan_level=2&scan_level=3&scan_level=4 HTTP/1.1" 500
2023-08-05T22:05:50.064706+02:00 sec-scan kat-octopoes[731]: [2023-08-05 22:05:50 +0200] [731] [ERROR] [h11_impl] Exception in ASGI application
2023-08-05T22:05:50.064793+02:00 sec-scan kat-octopoes[731]: Traceback (most recent call last):
2023-08-05T22:05:50.064840+02:00 sec-scan kat-octopoes[731]:   File "/opt/venvs/kat-octopoes/lib/python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 407, in run_asgi
2023-08-05T22:05:50.064884+02:00 sec-scan kat-octopoes[731]:     result = await app(  # type: ignore[func-returns-value]
2023-08-05T22:05:50.064954+02:00 sec-scan kat-octopoes[731]:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-08-05T22:05:50.064992+02:00 sec-scan kat-octopoes[731]:   File "/opt/venvs/kat-octopoes/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
                                                                                                                                                                                                                   21311,1       97%
```

Some kind of authenticationproblem?

--- a/boefjes/packaging/deb/data/etc/kat/katalogus.gunicorn.conf.py
+++ b/boefjes/packaging/deb/data/etc/kat/katalogus.gunicorn.conf.py
@@ -1,4 +1,2 @@
-import multiprocessing
-
-workers = multiprocessing.cpu_count() * 2 + 1
+workers = 3
 bind = ["127.0.0.1:8003"]

--- a/bytes/packaging/deb/data/etc/kat/bytes.gunicorn.conf.py
+++ b/bytes/packaging/deb/data/etc/kat/bytes.gunicorn.conf.py
@@ -1,4 +1,2 @@
-import multiprocessing
-
-workers = multiprocessing.cpu_count() * 2 + 1
+workers = 3
 bind = ["127.0.0.1:8002"]

--- a/keiko/packaging/deb/data/etc/kat/keiko.gunicorn.conf.py
+++ b/keiko/packaging/deb/data/etc/kat/keiko.gunicorn.conf.py
@@ -1,5 +1,2 @@
-"""Keiko gunicorn configuration file."""
-import multiprocessing
-
-workers = multiprocessing.cpu_count() * 2 + 1
+workers = 2
 bind = ["127.0.0.1:8005"]

--- a/octopoes/packaging/deb/data/etc/kat/octopoes.gunicorn.conf.py
+++ b/octopoes/packaging/deb/data/etc/kat/octopoes.gunicorn.conf.py
@@ -1,4 +1,2 @@
-import multiprocessing
-
-workers = multiprocessing.cpu_count() * 2 + 1
+workers = 4
 bind = ["127.0.0.1:8001"]

